changelog.d/gh-5506.fixed

```diff
@@ -0,0 +1 @@
+Scala/others: Added a fix allowing percolation of name information from class parameters
```

Comment:
You could elaborate the what/why more here. You could refer to the same example you used in the PR comment:
```Scala
class ExampleClass(val x: TypeName) {
}
```
But you don't need to repeat yourself, if something is explained in the changelog entry you could omit details in the PR comment.